tests/conftest.py

```diff
@@ -352,6 +352,11 @@ def rootfs_fxt(request, record_property):
 guest_kernel_linux_5_10 = pytest.fixture(
     guest_kernel_fxt, params=kernel_params("vmlinux-5.10*")
 )
+# Use the unfiltered selector, since we don't officially support 6.1 yet.
```

Comment:
nit: the "yet" sounds to me like we should add a TODO to use kernel_params when we officially start supporting 6.1.

Reply:
added the TODO